core/src/test/scala/org/apache/spark/ui/StagePageSuite.scala

```diff
@@ -78,6 +78,16 @@ class StagePageSuite extends SparkFunSuite with LocalSparkContext {
         shuffleLocalBytesRead = 1L,
         shuffleReadBytes = 1L,
         shuffleReadRecords = 1L,
+        shuffleCorruptMergedBlockChunks = 2L,
+        shuffleFallbackCount = 2L,
+        shuffleMergedRemoteBlocksFetched = 1L,
+        shuffleMergedLocalBlocksFetched = 1L,
+        shuffleMergedRemoteChunksFetched = 1L,
+        shuffleMergedLocalChunksFetched = 1L,
+        shuffleMergedRemoteBytesRead = 1L,
+        shuffleMergedLocalBytesRead = 1L,
+        shuffleRemoteReqsDuration = 1L,
+        shuffleMergedRemoteReqsDuration = 1L,
```

Comment:
Any particular reason these are set to `1` ?
Particularly when `shuffleMergersCount` == 0 ?

Reply:
We didn't put much thought into it. But `shuffleMergerCount` can be 0 because this stage may not push but this stage can still read shuffle data generated by the previous stage.

Reply:
nit: given inputBytes > 0, no likely :-) But that is not relevant to this test !